src/Concordium/Client/Types/Contract/Info.hs

```diff
@@ -270,6 +272,31 @@ data ContractStateV0
   | Cs0JSON !AE.Value
   deriving (Eq, Show)
 
+-- |Convert an `InstanceInfo` instance into a `ContractInfo` instance.
```

Comment:
There is a crucial point that is not mentioned here. namely what is done with the schema.

Please elaborate.